app/components/markdown_editor_component/view.html.erb

```diff
@@ -17,7 +17,7 @@
 
       <%= f.govuk_submit preview_button_translation, secondary: true,  name: "route_to", value: "preview", class: "app-markdown-editor__preview-button","data-ajax-markdown-trigger": true %>
 
-      <details class="govuk-details" data-module="govuk-details">
+      <details class="govuk-details">
```

Comment:
@DavidBiddle could we not use https://govuk-components.netlify.app/components/details/ so we dont have these govuk component hardcoded anywhere?

Reply:
Good shout - dealt with in a separate PR because I'm not sure how long this one will be open https://github.com/alphagov/forms-admin/pull/744

Reply:
Might be worth rebasing this PR now so we shouldn't need to worry about this tweak going forward.